modules/distribution/src/repository/resources/dbscripts/apimgt/oracle_rac.sql

```diff
@@ -133,21 +135,21 @@ CREATE TABLE IDN_OAUTH2_RESOURCE_SCOPE (
             SCOPE_ID INTEGER NOT NULL,
             TENANT_ID INTEGER DEFAULT -1,
             PRIMARY KEY (RESOURCE_PATH),
-            FOREIGN KEY (SCOPE_ID) REFERENCES IDN_OAUTH2_SCOPE (SCOPE_ID))
+            FOREIGN KEY (SCOPE_ID) REFERENCES IDN_OAUTH2_SCOPE (SCOPE_ID) ON DELETE CASCADE)
 /
 CREATE TABLE IDN_SCIM_GROUP (
-            ID INTEGER,
-            TENANT_ID INTEGER NOT NULL,
-            ROLE_NAME VARCHAR2(255) NOT NULL,
+			ID INTEGER,
+			TENANT_ID INTEGER NOT NULL,
+			ROLE_NAME VARCHAR2(255) NOT NULL,
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04